src/components/Card.js

```diff
@@ -21,11 +26,9 @@ export default class Card {
 
   _changeLike() {
     this._cardLikeElement.classList.toggle("card__like-button_active");
-  }
-
-  _deleteCard() {
-    this._cardElement.remove();
-    this._cardElement = null;
+    this._cardLikeElement.classList.contains("card__like-button_active")
+      ? api.likeCard(this.getCardData())
```

Comment:
You need to `catch` possible errors at the very end of any server request.

**Please, correct it everywhere**

Reply:
corrected